app/views/components/language_selection.scala.html

```diff
@@ -25,7 +25,7 @@
             id = Some(s"$key-switch"),
             url = s"${langToCall(key)}",
             value = Some(key.capitalize),
-            dataAttributes = Some(Map("journey-click" -> s"Language:Click:${if(key == "cymraeg") "Welsh" else key.capitalize}"))
+            dataAttributes = Some(Map("journey-click" -> s"link - click:LanguageSelection:${if(key == "cymraeg") "Welsh" else key.capitalize}"))
```

Comment:
Can you modify this to "link - click:Corporation Tax language select:"

Reply:
I've now made the amendment and re-checked the tests.